cmd/cluster-agent/api/v1/kubernetes_metadata.go

```diff
@@ -67,14 +67,14 @@ func getNodeMetadata(w http.ResponseWriter, r *http.Request, wmeta workloadmeta.
 	var dataBytes []byte
 	nodeName := vars["nodeName"]
 
-	nodeEntity, err := wmeta.GetKubernetesNode(nodeName)
+	nodeMetadata, err := wmeta.GetKubernetesMetadata(fmt.Sprintf("nodes//%s", nodeName))
```

Comment:
Discussed with @adel121 to try to provide helpers so that callers to `GetKubernetesMetadata` don't need to know the format of the ID.